devel/happy/Portfile

```diff
@@ -4,27 +4,27 @@ PortSystem          1.0
 PortGroup           haskell_stack 1.0
 
 name                happy
-version             1.19.12
-revision            1
+version             1.20.0
+revision            0
 categories          devel haskell
 maintainers         nomaintainer
 license             GPL-3
 platforms           macosx
 homepage            https://github.com/simonmar/happy
 
 description         A parser generator for Haskell
-long_description    Happy is a parser generator for Haskell. Given a\
-                    grammar specification in BNF, Happy generates\
-                    Haskell code to parse the grammar. Happy works in\
+long_description    Happy is a parser generator for Haskell. Given a \
+                    grammar specification in BNF, Happy generates \
+                    Haskell code to parse the grammar. Happy works in \
                     a similar way to the yacc tool for C.
```

Comment:
please don't make unneeded white-space changes as they clutter the diff to see the actual functional changes

Reply:
Removed.